WordPress/WordPressUITests/WordPressUITests.xctestplan

```diff
@@ -15,7 +15,8 @@
       "containerPath" : "container:WordPress.xcodeproj",
       "identifier" : "1D6058900D05DD3D006BFB54",
       "name" : "WordPress"
-    }
+    },
+    "testRepetitionMode" : "retryOnFailure"
```

Comment:
There are four modes. Here's my thoughts on when to use them:

1. _None_. Do not repeat the tests
2. _Until Failure_. Runs the tests until one fails, up to three times. Useful to discover flaky tests.
3. _Retry on Failure_. Will retry the tests that failed up to three times. Helpful to work around flaky tests that you haven't had the time to fix yet.
4. _Up Until Maximum Repetition_. Will run the tests three times (the number doesn't seem configurable yet) and report the success ratio. Useful to gauge the stability of your test suite.

I think "Retry on Failure" is the right option for us. In the future, it might be cool to explore using Until Failure on a dedicated pipeline running on a schedule to hammer the suite and discover flakes.